### PR TITLE
Various suggestions and code improvements, fixed one possible mistake

### PR DESCRIPTION
**Pull request by danielniccoli**
Please review 5cba5c2. The two lines would have never been executed. I don't fully understand the code yet, but to me it looks like it was indented once too much.

Merge the other commits as you please.

### CODE REVIEW

**Comment by andrewgodwin** [comment]
The unindent is fine, but if you wrap things onto multiple lines can you use the form:

```foo(
    x,
    y,
    z,
)```

rather than a single indented new line?

**Comment by danielniccoli** [comment]
Sure. But could you point to a line of code as an example? I am not 100 % sure what you mean with the single indented new line.

File: asgi_ipc.py [hunk 1]
```diff
@@ -40,16 +42,20 @@ def __init__(self, prefix="asgi", expiry=60, group_expiry=86400, capacity=10, ch
         )
         self.thread_lock = threading.Lock()
         self.prefix = prefix
-        self.channel_store = MemoryDict("/%s-chan" % self.prefix, size=channel_memory)
+        self.channel_store = MemoryDict("/%s-chan" % self.prefix,
+                                        size=channel_memory)
```
Comment (andrewgodwin) [inline asgi_ipc.py]: This should be indented differently - like this:

```
MemoryDict(
    "/%s-chan" % self.prefix,
    size=channel_memory,
)
```

**Comment by danielniccoli** [comment]
Is this okay?

**Comment by andrewgodwin** [comment]
Yup, thanks!

**Pull request merged**